instruments/tests/test_named_struct.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Module containing tests for named structures.
+"""
+
+# IMPORTS ####################################################################
+
+from __future__ import absolute_import
+
+from instruments.named_struct import (
+    Field, StringField, Padding, NamedStruct
+)
+
+# TESTS ######################################################################
+
+# pylint: disable=no-member,protected-access,blacklisted-name,missing-docstring
+
+def test_named_struct_roundtrip():
```

Comment:
That's awesome, love that it works out for this case. That may help us write other unit tests in the future, too. For this PR, maybe it makes sense to use a string strategy for ``StringField`` instances?